controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -306,13 +308,24 @@ func (r *DataScienceClusterReconciler) reportError(err error, instance *dsc.Data
 	return instance
 }
 
+var configMapPredicates = predicate.Funcs{
+	UpdateFunc: func(e event.UpdateEvent) bool {
+		fmt.Printf("Got update event for resource %v: %v", e.ObjectNew.GetObjectKind(), e.ObjectNew.GetName())
```

Comment:
just double checking if you intend to leave this log in. I think it should be fine, but wanted to double check.

Reply:
yeah, we should remove/comment it out here.

Reply:
Removed